README.md

```diff
@@ -51,9 +51,10 @@ you can then communicate with the Extension using the BladeSigner object using t
 
 | API                                                        | Description                                                      |
 |:-----------------------------------------------------------| :--------------------------------------------------------------- |
+| `bladeSigner.createSession(network:HederaNetwork,dAppCode:number)`| Optional params. Create session with Blade extension.                                 |
```

Comment:
dAppCode is not a number its a string. Provide examples for network and dAppCode.